ui2/src/pages/custody/Account.tsx

```diff
@@ -15,6 +15,7 @@ import { usePartyName } from '../../config';
 import StripedTable from '../../components/Table/StripedTable';
 import { ServicePageProps } from '../common';
 import { ArrowLeftIcon } from '../../icons/icons';
+import {AllocationAccountRule} from "@daml.js/da-marketplace/lib/Marketplace/Rule/AllocationAccount/module";
```

Comment:
did you run `yarn format`? should have caught the missing spaces between the `{}`

Reply:
oh and a nit pick - could you remove the `/module` at the end of the import? I've noticed VS Code sometimes puts that in automatically, and it's not required

Reply:
I'll fix both points on the next commit - well spotted!! 👀